lib/build.gradle

```diff
@@ -26,12 +26,12 @@ ext {
 
 apply from: 'publish.gradle'
 android {
-    compileSdk 32
+    compileSdk 33
 
     defaultConfig {
 
-        minSdk 21
-        targetSdk 32
+        minSdk 33
+        targetSdk 33
```

Comment:
Is this change needed to fix the test ?

We agree in #25 that it was not the time yet to update the minimum supported version.